draft-cutler-httpbis-partitioned-cookies.md

```diff
@@ -102,7 +105,7 @@ The following could also be added as an additional step to section 5.4:
 {:quote}
 > 1.  If the cookie-attribute-list does not contain an attribute with an attribute-name of "Partitioned", set partition-key to null.<br><br>
       If the cookie-attribute-list does contain an attribute with an attribute-name of "Partitioned" and the secure-only-flag is false, abort these steps and ignore the cookie entirely.<br><br>
-      Otherwise, set partition-key to cookie-partition-key defined in section 5.X.X.
+      If the cookie being set matches the top-level site, check if there is a cross-site ancestor present and use that value to, set partition-key to cookie-partition-key defined in section 5.X.X.
```

Comment:
This line's change is not needed since we already include the ancestor-chain bit in the definition of computing the partition key.